frontend/src/components/layout/Header/UserProfile.tsx

```diff
@@ -14,7 +14,7 @@ const StyleDropDown = styled(NavDropdown)`
   font-size: 14px;
   .dropdown-menu {
     width: 300px;
-    height: 120px;
+    height: 150px;
```

Comment:
Do we need a static height here? could just set some padding here and just let it grow to accommodate.

Reply:
I had that originally, but it kept leaving a little bit of white trim underneath the blue footer. Will try playing around with it again though!